modules/fluid_properties/doc/content/source/fluidproperties/ExternalFluidPropertiesSubmodules.md

```diff
@@ -0,0 +1,20 @@
+# External Fluid Properties Submodules
```

Comment:
This page is good, but I don't see a link to it anywhere. Did I miss it? I would think that it should go on the fluid properties module page.

Reply:
the page is available in the source and syntax dumps. We dont usually link to new objects specifically?

Reply:
https://mooseframework.inl.gov/docs/PRs/27027/site/source/fluidproperties/ExternalFluidPropertiesSubmodules.html

Reply:
It's in the source index, but I don't think it's in the syntax index, right? Imagine you're a user that is interested in getting air properties. You first browse the syntax index and you see an air fluid properties, so you click on that page, but it tells you nothing about how to get that class. Maybe after that you try looking at the fluid properties module page to see if it talks about it, but you don't find anything. You have no indication that what you need to be doing is going to the source index and looking for a page called "ExternalFluidPropertiesSubmodules".

Reply:
Those other pages live on the submodules we can only link to those once moose has been updated to have this page first

Reply:
By syntax page I just mean the object dump organized by syntax rather than by system/folder

Reply:
> Those other pages live on the submodules we can only link to those once moose has been updated to have this page first

I'm confused - I'm able to go to https://mooseframework.inl.gov/docs/PRs/27027/site/fluidproperties/AirSBTLFluidProperties.html for example.

> By syntax page I just mean the object dump organized by syntax rather than by system/folder

I'm referring to https://mooseframework.inl.gov/docs/PRs/27027/site/syntax/index.html to be clear. I don't think the page would appear here without being located in `syntax/` and having a syntax.

Reply:
> I'm confused - I'm able to go to

yes but it comes from the air git submodule. So to add a link there we would need to make a PR to the submodule not to moose

> I don't think the page would appear here without being located in syntax/ and having a syntax.

yeah no I would not want to do that. FluidProperties/ is still the syntax and we dont want to change that. 
By syntax I mean this page
https://mooseframework.inl.gov/docs/PRs/27027/site/syntax/FluidProperties/index.html

Reply:
> yes but it comes from the air git submodule. So to add a link there we would need to make a PR to the submodule not to moose

I understand what you're saying now, thanks! So if the plan is to update all of the submodules with links to this page, then that's good. Best would be to additionally have a link on the module page - I can add that if you like.